Added details to retrieve custom script output

amoghnatu opened the pull request:
1. Added the details of PowerShell cmdlets to use for retrieving the output of the script that is executed as part of the CustomScript Execution. 

2. Also added a note on how to use the same when running them inside an automation runbook of type workflow as the cmdlet doesn't return straightforward output while running in runbook of type workflow. Spent a lot of time myself to figure this out.

azurecla commented:
Hi __@Taojunshen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-jutao). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

amoghnatu committed:
Added details to retrieve custom script output

Added the details of PowerShell cmdlets to use for retrieving the output of the script that is executed as part of the CustomScript Execution. Also added a note on how to use the same when running them inside an automation runbook of type workflow as the cmdlet doesn't return straightforward output while running in runbook of type workflow.
